pulp_container/app/viewsets.py

```diff
@@ -796,6 +796,16 @@ class ContainerRepositoryVersionViewSet(RepositoryVersionQuerySetMixin, Reposito
                     "has_repo_attr_model_or_obj_perms:container.view_containerrepository",
                 ],
             },
+            {
+                "action": ["repair"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_repo_attr_model_or_obj_perms:container.sync_containerrepository",
```

Comment:
@mdellweg @goosemania @dkliban can you weigh in here?
I thought to use `modify_content_containerrepository` but none of  the actions assigned to that permission bring in RA.
I can introduce a new permission `container.repair_content_containerrepository` but i am trying to avoid the migration

Reply:
looks like a reasonable choice to me.

Reply:
also, i have on purpose non added `repair` for the container-push repo because all of the content that comes in it comes through the upload.

Reply:
I think your choice is reasonable, though it brings me back to the topic of how to make it understandable for users which perms they need for what if they work with pulp's default policies.
Should the default policies become a part of the generated documentation?

Reply:
I agree that this change is rather implicit than explicit.
The only place where the remissions are 'documented' in some way is here https://github.com/pulp/pulp_container/blob/master/pulp_container/app/models.py#L372
It would be great if we can have default policy documented